book-1-orientation/chapters/STUDENT_EXERCISES_ADONET.md

```diff
@@ -71,17 +71,18 @@
     }
     ```
      Create menu options to allow the user to perform the following tasks:
-    1. Display all students.
-    1. Display all instructors.
-    1. Display all exercises.
-    1. Display all cohorts.
-    1. Search students by last name.
-    1. Create a new cohort.
-    1. Create a new student and assign them to an existing cohort.
-    1. Create a new instructor and assign them to an existing cohort.
-    1. Display all students in a given cohort.
-    1. Move an existing student to another existing cohort.
-    1. List the exercises for a given student.
+    1. Display all dogs.
+    1. Display all walkers.
+    1. Display all neighborhoods.
+    1. Display all walks.
+    1. Display all owners.
+    1. Search walkers by last name.
```

Comment:
Walkers only have a `Name` property. Maybe we can change this to just search by name, or search by neighborhood name